src/main/java/org/killbill/billing/osgi/api/ServiceRegistry.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2010-2014 Ning, Inc.
+ * Copyright 2014-2020 Groupon, Inc
+ * Copyright 2020-2022 Equinix, Inc
+ * Copyright 2014-2022 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.osgi.api;
+
+public interface ServiceRegistry {
```

Comment:
Could we rename this to `ServiceLocationRegistry` or `ServiceDiscoveryRegistry` -> the name `ServiceRegistry` in the OSGI context is already so overloaded!

Reply:
Agreed 🙈

Renamed to `ServiceDiscoveryRegistry`.